### ISSUE TITLE
java.lang.RuntimeException: DeviceUtils.java

### ISSUE THREAD

**Issue by Digisha** [intercom/intercom-android#218]
We are getting DeadObjectException in our app.

## Version info
  - Intercom for Android version: 1.1.21
  - Android OS version: 6.0.1, 5.0
  - Android Device: SM-G920R4, SM-G900R4, 

## Logs
```
	java.lang.RuntimeException: Package manager has died
37	at android.app.ApplicationPackageManager.getPackageInfo(ApplicationPackageManager.java:133)
38	at io.intercom.android.sdk.commons.utilities.DeviceUtils.getAppVersion(DeviceUtils.java:84)
39	at io.intercom.android.sdk.api.Api.configureRestAdapter(Api.java:109)
40	at io.intercom.android.sdk.api.Api.createIntercomApiInterface(Api.java:96)
41	at io.intercom.android.sdk.api.Api.<init>(Api.java:84)
42	at io.intercom.android.sdk.Bridge.<init>(Bridge.java:52)
43	at io.intercom.android.sdk.Bridge.initWithAppCredentials(Bridge.java:75)
44	at io.intercom.android.sdk.RealIntercom.<init>(RealIntercom.java:57)
45	at io.intercom.android.sdk.Intercom.initialize(Intercom.java:45)
46	at com.mobiledefense.lean.intercom.IntercomGatewayImpl.initialize(IntercomGatewayImpl.java:29)
47	at com.mobiledefense.lean.features.LeanFeatureListener.onFeatureChange(LeanFeatureListener.java:60)
48	at com.mobiledefense.lean.features.LeanFeatureListener.init(LeanFeatureListener.java:36)
49	at com.mobiledefense.base.module.FeatureChangeListener.initialize(FeatureChangeListener.java:10)
50	at com.mobiledefense.MobileDefenseApplication.registerFeatureChangeListeners(MobileDefenseApplication.java:182)
51	at com.mobiledefense.MobileDefenseApplication.onCreate(MobileDefenseApplication.java:108)
52	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1020)
53	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5010)
54	at android.app.ActivityThread.access$1600(ActivityThread.java:172)
55	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1482)
56	at android.os.Handler.dispatchMessage(Handler.java:102)
57	at android.os.Looper.loop(Looper.java:145)
58	at android.app.ActivityThread.main(ActivityThread.java:5835)
59	at java.lang.reflect.Method.invoke(Native Method)
60	at java.lang.reflect.Method.invoke(Method.java:372)
61	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1388)
62	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1183)
63	Caused by: android.os.DeadObjectException
64	at android.os.BinderProxy.transactNative(Native Method)
65	at android.os.BinderProxy.transact(Binder.java:496)
66	at android.content.pm.IPackageManager$Stub$Proxy.getPackageInfo(IPackageManager.java:1961)
67	at android.app.ApplicationPackageManager.getPackageInfo(ApplicationPackageManager.java:128)
68	... 25 more
```

**Comment by ronocod** [comment]
Thanks for reporting this. We've merged a fix which will be in our next update. I'll add a comment here when it's released :)

**Comment by Digisha** [comment]
@ronocod Thank you for the update!

**Comment by ronocod** [comment]
We've released version [3.0.0](https://github.com/intercom/intercom-android/releases/tag/3.0.0) which fixes this. If you still see this issue after updating to 3.0.0 please let us know.
